plugins/tekton/src/components/PipelineRunLogs/TaskStatusStepper.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { memo, useState } from 'react';
+import useInterval from 'react-use/lib/useInterval';
+
+import {
+  CircularProgress,
+  createStyles,
+  makeStyles,
+  StepButton,
+  StepIconProps,
+  Theme,
+} from '@material-ui/core';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Stepper from '@material-ui/core/Stepper';
+import Typography from '@material-ui/core/Typography';
+import Cancel from '@material-ui/icons/Cancel';
+import Check from '@material-ui/icons/Check';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import classNames from 'classnames';
+
+import { ComputedStatus } from '@janus-idp/shared-react';
+
+import { TaskStep } from '../../utils/taskRun-utils';
+import { calculateDuration } from '../../utils/tekton-utils';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    titleContainer: {
+      display: 'flex',
+      alignItems: 'center',
+      gap: theme.spacing(1),
+    },
+    closeButton: {
+      position: 'absolute',
+      right: theme.spacing(1),
+      top: theme.spacing(1),
+      color: theme.palette.grey[500],
+    },
+    labelWrapper: {
+      display: 'flex',
+      flex: 1,
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+    },
+    stepWrapper: {
+      width: '100%',
+    },
+  }),
+);
+
+const StepTimeTicker = ({ step }: { step: TaskStep }) => {
+  const [time, setTime] = useState('');
```

Comment:
nit: good to be consistent in some places in this PR itself we use `React.useState` and here and in some places we have `useState`